pub_sub/go/sdk/checkout/app.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"context"
+	"log"
+	"strconv"
+	"time"
+
+	dapr "github.com/dapr/go-sdk/client"
+)
+
+var (
+	PUBSUB_NAME  = "order_pub_sub"
+	PUBSUB_TOPIC = "orders"
+)
+
+func main() {
+	client, err := dapr.NewClient()
+	if err != nil {
+		panic(err)
+	}
+	defer client.Close()
+	ctx := context.Background()
+	for i := 1; i <= 10; i++ {
+		order := "{\"orderId\":" + strconv.Itoa(i) + "}"
+
+		// Dapr subscription routes orders topic to this route
```

Comment:
I'd expect this comment only in a subscriber, like order-processor

Reply:
in the http version this is the comment which seems better:

		// Publish an event using Dapr pub/sub